ewkcoffea/modules/objects_wwz.py

```diff
@@ -161,47 +162,86 @@ def get_topmva_score_mu(events, year):
     score = ak.unflatten(score,counts)
     return score
 
-def is_veto_run3_2lep_ele(ele):
+#################### Run 2 and Run 3 TnP IDs #######################
+
+def is_loose_run2_ele(ele):
```

Comment:
Is this only used for TnP stuff? If so could you add something to the name (e.g. `tnp` somewhere) to specify so that we don't get it confused with our analysis selection?

Reply:
added tnp tag